dxr/plugins/js/indexers.py

```diff
@@ -0,0 +1,105 @@
+from os.path import basename, dirname, relpath, join, exists
+import json
+import subprocess
+
+from funcy import imap
```

Comment:
Would itertools.imap work instead? I rather don't trust funcy; I've found some real inefficiencies in there, and I would like to get rid of it eventually.